router/authentication/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package authentication implements the types and functions
+// used to load and validate user authentication
+package authentication
```

Comment:
Empty package?

Reply:
If you already saw a comment by me on this, please ignore it. It looked to me like my first attempt didn't send.

Reply:
This is a convention for Go packages. The package documentation goes into a file called `doc.go` which has nothing else. See, for example, net/http https://cs.opensource.google/go/go/+/master:src/net/http/doc.go